schema.go

```diff
@@ -148,6 +149,9 @@ func mapStruct(dst *Struct, src *ast.StructType) error {
 			return fmt.Errorf("colfer: missing name for field %d", i)
 		}
 		field.Name = f.Names[0].Name
+		if f.Tag != nil {
+			field.Tag = strings.Trim(f.Tag.Value, "`")
+		}
```

Comment:
Missing language check. Now it silently ignores these values in many cases.

Reply:
The idea is that we could use it to annotate fields in other languages too, potentially. Which is why I kept it. Also, I wanted to start using tags for nullable-types or other ways to control code generation.

Reply:
You could transfer these tags to Golang as well, as necessary. And then there's C# which has Attributes. Things like that.

Reply:
What I mean is you can't read and ignore a value. No error or warning means you got it. Otherwise the user is left hanging why his tags are ignored.